interface/client/appStart.js

```diff
@@ -56,12 +56,7 @@ mistInit = function () {
                 }
             });
 
-        // Sets browser as default tab if:
-        // 1) there's no record of selected tab
-        // 2) data is corrupted (no saved tab matches localstore)
-        if (!LocalStore.get('selectedTab') || !Tabs.findOne(LocalStore.get('selectedTab'))) {
-            LocalStore.set('selectedTab', 'wallet');
-        }
+        LocalStore.set('selectedTab', 'wallet');
```

Comment:
With this change, if the user has another tab selected and quits Mist, it'll always select the wallet.

It should keep the previous tab.

Reply:
i guess the logic was correct after all; reverted.